tls/s2n_config.c

```diff
@@ -1138,3 +1139,45 @@ int s2n_config_set_cert_validation_cb(struct s2n_config *config, s2n_cert_valida
 
     return S2N_SUCCESS;
 }
+
+int s2n_config_get_supported_groups(struct s2n_config *config, uint16_t *groups, uint16_t groups_count_max,
```

Comment:
Why is this API on the config? 🤔 I don't think it actually needs the config at all except to retrieve the security policy.

Reply:
Yes correct - it only needs the config to get the security policy. We could instead make this API take the security policy string. However, if we were to some day add a security policy builder that returns a `struct s2n_security_policy *` instead of a string, the string version of the API would not be compatible with that.